orbit/cmd/orbit/orbit.go

```diff
@@ -494,6 +522,22 @@ func main() {
 		}
 		g.Add(r.Execute, r.Interrupt)
 
+		updatedChan := make(chan struct{})
+		go retry(constant.OrbitFlagsInterval*time.Second, true, updatedChan, func() bool {
```

Comment:
Actually here you can re-use the `g.Add` approach.

And then you don't need to worry about manually doing the kill.
See the update example:
https://github.com/fleetdm/fleet/blob/9ef0972bd500cabbc58ffb0a0c6f2d4b038dd08f/orbit/cmd/orbit/orbit.go#L295
And then it just returns whenever a new update is detected:
https://github.com/fleetdm/fleet/blob/9ef0972bd500cabbc58ffb0a0c6f2d4b038dd08f/orbit/pkg/update/runner.go#L79-L102

Reply:
Ah, I see..so this will probably look like adding a Flags Updater, and adding that to same run group?

Reply:
Exactly.

Reply:
Cool, updated to use the run group approach